bin/update_truss_container_fs_for_tests.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# This script generates truss/test_data/truss_container_fs from
+# truss/test_data/test_truss.
+#
+# Idea is to get access to representative /app folder content of a sample truss
+# for testing purposes.
+#
+# Contents of truss_container_fs should then be checked into git.
+
+import subprocess
+from pathlib import Path
+
+ROOT = str(Path(__file__).parent.parent.resolve())
+
+
+if __name__ == "__main__":
+    subprocess.run(["truss", "run-image", "truss/test_data/test_truss"], cwd=ROOT)
+    ps_output = subprocess.check_output(
```

Comment:
Any reason we don't use python_on_whales here to interact?

Reply:
We need this for running the truss command anyway. I found that this is much easier for a script. With python-on-whales I have to check for how everything is exposed in pyhon-on-whales API. We can revisit if this gets complex or if we encounter any maintenance issues.

Reply:
I don't know if I'm necessarily advocating for python_on_whales, but more for consistency across the codebase.  Not a blocker either way.